__tests__/app.test.js

```diff
@@ -203,5 +249,67 @@ describe("app", () => {
           });
       });
     });
+    describe("POST /api/articles/:article_id/comments", () => {
+      describe("status: 404", () => {
+        it("404: responds with correct message when sent an invalid user", () => {
+          const testNewComment = {
+            username: "",
+            body: "New Comment",
+          };
+
+          return request(app)
+            .post("/api/articles/4/comments")
+            .send(testNewComment)
+            .expect(404)
+            .then(({ body }) => {
+              expect(body.msg).toBe("Author Not Found");
+            });
+        });
+      });
+      describe("status: 400", () => {
+        it("404: responds with correct message for valid but non-existent article_id", () => {
+          const testNewComment = {
+            username: "butter_bridge",
+            body: "New Comment",
+          };
+
+          return request(app)
+            .post("/api/articles/4321/comments")
+            .send(testNewComment)
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toBe("Article Does Not Exist");
+            });
+        });
+        it("400: responds with correct message when sent a invalid parametric endpoint", () => {
+          const testNewComment = {
+            username: "butter_bridge",
+            body: "New Comment",
+          };
+
+          return request(app)
+            .post("/api/articles/four/comments")
+            .send(testNewComment)
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toBe("Bad Request");
+            });
+        });
+        it("400: responds with correct message when sent an object without a comment", () => {
+          const testNewComment = {
+            username: "butter_bridge",
+            body: "",
+          };
+
+          return request(app)
+            .post("/api/articles/4/comments")
+            .send(testNewComment)
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toBe("Invalid Comment Format");
+            });
+        });
+      });
```

Comment:
Can you add another test here for 201 - ignores invalid properties? 🤔

Reply:
No problem :)